frontend/src/data/report-service.ts

```diff
@@ -1,5 +1,5 @@
 import type { MediaType } from "~/constants/media"
-import type { ReportReason } from "~/constants/content-report"
+import { ReportReason } from "~/constants/content-report"
```

Comment:
It's only being used as a type, so why this change?

Reply:
Whoops, just accidentally left in when I removed the frontend adapter code in favour of having the API accept the new term at the serialiser.

Reply:
Fixed in https://github.com/WordPress/openverse/pull/3006/commits/c47176e7e2dd9dc16bef3e969690305116919514